test/k8s-integration/main.go

```diff
@@ -212,7 +212,7 @@ func handle() error {
 		if err != nil {
 			return fmt.Errorf("failed to download Kubernetes source: %v", err)
 		}
-		err = buildKubernetes(testDir, "WHAT=test/e2e/e2e.test")
+		err = buildKubernetes(testDir, "quick-release")
```

Comment:
can you add a "TODO" to parallelize these builds since they'll take a long time now

Reply:
Thinking about this, it may not make sense to parallelize k8s builds because each quick-release is already parallelized you can check htop to confirm). You might save some time on the go builds for certain components that are not, but then suffer from the context switching. 

This is also why my desktop was dying earlier - I was parallelizing two builds.

Reply:
I changed the PR so it builds kubectl specificlly (I figured out how). Currently testing to see if this solves it.

Reply:
ping me when ready for review